store/views.py

```diff
@@ -12,9 +14,14 @@ def index(request):
 
 def bookDetailView(request, bid):
     template_name = 'store/book_detail.html'
+    book=Book.objects.get(id=bid)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.